packages/consumption/test/modules/requests/itemProcessors/readAttribute/ReadAttributeRequestItemProcessor.test.ts

```diff
@@ -353,16 +353,16 @@ describe("ReadAttributeRequestItemProcessor", function () {
         });
 
         test("returns an error when the given Attribute id belongs to a peer Attribute", async function () {
```

Comment:
While you're at it, write "peer shared Attributes" here instead of "peer Attributes" please.